src/bootstrap/src/core/sanity.rs

```diff
@@ -32,13 +32,7 @@ pub struct Finder {
 // it might not yet be included in stage0. In such cases, we handle the targets missing from stage0 in this list.
 //
 // Targets can be removed from this list once they are present in the stage0 compiler (usually by updating the beta compiler of the bootstrap).
-const STAGE0_MISSING_TARGETS: &[&str] = &[
```

Comment:
Suggestion: can you please keep the

```
// just a dummy comment so the list doesn't get onelined
```

comment to inhibit the oneline formatting?